core/org.openjdk.jmc.flightrecorder/src/main/resources/org/openjdk/jmc/flightrecorder/stacktrace/messages/internal/messages.properties

```diff
@@ -36,3 +36,4 @@ STACKTRACE_LINE_NUMBER=Line Number
 STACKTRACE_METHOD=Method
 STACKTRACE_PACKAGE=Package
 STACKTRACE_UNCLASSIFIABLE_FRAME=~ UNCLASSIFIABLE ~
+STACKTRACE_UNCLASSIFIABLE_FRAME_DESC=Unclassified means the stacktrace reached the stackdepth limit
```

Comment:
Lacks a newline at the end of the last property.

Reply:
corrected